frontend/src/component/featureTypes/FeatureTypeEdit/FeatureTypeForm/FeatureTypeForm.tsx

```diff
@@ -79,14 +79,19 @@ export const FeatureTypeForm: VFC<FeatureTypeFormProps> = ({
                 throw new Error('No feature toggle type loaded');
 
             const value = doesntExpire ? 0 : lifetime;
-            await updateFeatureTypeLifetime(featureType?.id, value);
+            await updateFeatureTypeLifetime(featureType.id, value);
```

Comment:
We've already verified that it's non-empty here, so it should be safe to remove the `?`